docs/topics/api/addons.rst

```diff
@@ -56,8 +55,9 @@ This endpoint allows you to search through public add-ons.
             random  Random ordering. Only available when no search query is
                     passed and when filtering to only return promoted add-ons.
             rating  Bayesian rating, descending.
-       recommended  Recommended add-ons above non-recommend add-ons. Only
-                    available combined with another sort - ignored on its own.
+       recommended  Promoted addons in the recommended category above
+                    non-recommend add-ons. Only available combined with another
```

Comment:
Not sure I understand the sentence here.

Reply:
https://github.com/mozilla/addons-server/pull/15470/commits/967ddaa4db2ba47b5c6c2ff2b351bed434974ef6